client.go

```diff
@@ -1647,6 +1644,10 @@ func (c *Client[TTx]) JobListTx(ctx context.Context, tx TTx, params *JobListPara
 // client, and can be used to add new ones or remove existing ones.
 func (c *Client[TTx]) PeriodicJobs() *PeriodicJobBundle { return c.periodicJobs }
 
+// Queues returns the currently configured set of queues for the client, and can
+// be used to add new ones.
+func (c *Client[TTx]) Queues() *queueBundle { return c.queues }
```

Comment:
I don't think we want to have an exported method that returns an unexported type like this. Whether or not it works, it looks bad/undiscoverable in docs.

I think the type either needs to be exported with minimal methods exposed on it, or else we need to return an exported interface type here.

Reply:
Ah yeah, makes sense. I'd make it unexported initially because I had to add a `TTx` generic parameter so it could internalize a client, and it was just a really ugly type. Now that that's gone, totally okay to export this.